web/source/keymanweb.js

```diff
@@ -2904,14 +2904,14 @@
     var n,Ln,lgCode;
```

Comment:
It seems that `lgCode` is still used within the function (lines 2921,2925). These references need refactoring as well.